src/routers/management/config.py

```diff
@@ -43,7 +78,7 @@ def update_config(
     )
 
 
-@r.get("/models")
+@r.get("/available_models", tags=["Model config"])
```

Comment:
part of models group? e.g.:
```suggestion
@r.get("/models/list", tags=["Model config"])
```